tests/apps/scout/test_scout_load_config.py

```diff
@@ -28,6 +29,28 @@ def test_validate_mt_bam(sample_dict):
     assert ind_obj.mt_bam == sample["mt_bam"]
 
 
+def test_validate_reviewer_alignment(sample_dict):
```

Comment:
I think you can use `@pytest.mark.parametrize`, https://docs.pytest.org/en/6.2.x/parametrize.html,  here to test any number of file with one general function and remove the other test

Reply:
Interesting, will look into it

Reply:
Sorry, don't know enough about pytests to implement this. If someone is willing to walk me through it I can try to implement it. Otherwise I suggest that we merge the code as is.

Reply:
👍

Reply:
Let’s go a head and not spend time on this unit test. It is good enough